### PR TITLE
Update and move tools

### PR DESCRIPTION
**Pull request by Brotenko**

### CODE REVIEW

**Review by ethteck — CHANGES_REQUESTED** [review]
LGTM except for two things:

codescan.py now won't work, as you need to refer to two directories up as the root dir
Before "to_rename" is opened in rename.py, can you change the comment to
`# Read Star Rod's output file (one rename per line, old and new, delimited by a space)` since you removed to_rename.txt?

File: tools/star_rod_enum_to_decomp.py [hunk 1]
```diff
@@ -34,11 +27,11 @@ def create_enum(file_content, prefix, ordering):
 
     for (key, value) in re.findall(r'(\S+)\s+=\s+(\S+)', file_content):
         if ordering:
-            key = '_'.join(re.sub(r'([A-Z])', r' \1', key).split()).replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP").replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
+            key = '_'.join(re.sub(r'([A-Z]{1,2})', r' \1', key).split()).replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
             key = prefix.upper() + '_{:<{width}}'.format(key, width=max_size + 2).upper()
             ret += "    " + key + " = 0x" + '{:>{fill}{width}}'.format(value, fill=0, width=8) + ",\n"
         else:
-            value = '_'.join(re.sub(r'([A-Z])', r' \1', value).split()).replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP").replace("N_P_C", "NPC").replace("__", "_").replace("-", "")
```
Comment (nanaian) [inline tools/star_rod_enum_to_decomp.py:41]: What happened to `.replace("H_P", "HP").replace("F_P", "FP").replace("J_P", "JP")`

Comment (Brotenko) [reply tools/star_rod_enum_to_decomp.py:41]: No longer needed as the regex can detect those properly now

**Comment by BowserSlug** [comment]
✅ This PR fixes 49 warnings!

**Review by ethteck — APPROVED** [review]

**Pull request merged**